test/extended/util/annotate/rules.go

```diff
@@ -52,6 +52,110 @@ var (
 		"[Skipped:ovirt]": {},
 		"[Skipped:gce]":   {},
 
+		// Tests that don't pass on disconnected, either due to requiring
+		// internet access for GitHub (e.g. many of the s2i builds), or
+		// because of pullthrough not supporting ICSP (https://bugzilla.redhat.com/show_bug.cgi?id=1918376)
+		"[Skipped:Disconnected]": {
```

Comment:
If you know easily, can you split the chunk here into two sections (one for ICSP pullthrough, one for s2i builds).  Anything explicitly ICSP helps in enabling it later

Reply:
Sure, done.  There's a few that overlap, e.g. they're failing on pulling the ruby image now but later on will fail to clone https://github.com/openshift/ruby-hello-world.git, but I've split them best I can.